bodhi/server/util.py

```diff
@@ -786,4 +806,17 @@ def pdc_api_get(pdc_api_url):
     """
     # There is no error_key specified because the error key is not consistent
     # based on the error message
-    return api_get(pdc_api_url, service_name='PDC')
+    return call_api(pdc_api_url, service_name='PDC')
+
+
+def greenwave_api_post(greenwave_api_url, data):
+    """
+    A wrapper to post request to Greenwave.
+    :param greenwave_api_url: a string of the URL to query
+    :param data: A dictionary of data to send along the request
+    :return: JSON of the API response or a RuntimeError
+    """
+    # There is no error_key specified because the error key is not consistent
+    # based on the error message
+    return call_api(greenwave_api_url, service_name='Greenwave', method='POST',
```

Comment:
**Please be VERY CAUTIOUS HERE!**
This code, via call_api, ASSUMES that any non-500 error returned a JSON results, which is DEFINITELY not the result with greenwave, errors are either HTML or plain text, see: https://pagure.io/greenwave/issue/37.